test/token/ERC20/ERC20.behavior.js

```diff
@@ -148,21 +148,6 @@ function shouldBehaveLikeERC20 (errorPrefix, initialSupply, initialHolder, recip
           });
         });
       });
-
-      describe('when the recipient is the zero address', function () {
```

Comment:
Since everything goes through the transfer, it cannot know if its coming from mint or burn, so `transfer` has a check to ensure that `from` and `to` cannot be the zero address, but if a user sends a mint to address zero it would received both `from` and `to` as the zero address. There is a test for that case on [ln 206](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/27c7090480c2dc26f361def7b4a62960a591f55b/test/token/ERC20/ERC20.test.js#L206)